src/Compiler/Utilities/range.fsi

```diff
@@ -121,6 +122,7 @@ type Range =
     member internal IsAdjacentTo: otherRange: Range -> bool
 
     /// The range where all values are zero
+    [<Obsolete("Use Range.range0 instead")>]
     static member Zero: range
```

Comment:
I guess there doesn't seem to be anything that currently relies on the presence of this member to satisfy a constraint like this that would be added, e.g., by anything that uses `LanguagePrimitives.GenericZero`?

```fsharp
'a when 'a : (static member Zero : 'a)
```

Given the discussion [here](https://github.com/dotnet/fsharp/pull/18610#discussion_r2107739342), it does seem like that would be ill-advised anyway (unless `Zero` were defined like `range0` instead).